src/services/resourceService.test.ts

```diff
@@ -102,10 +105,10 @@ describe("Resource Service", () => {
     const service = new ResourceService(sls, options);
     const deploymentString = await service.listDeployments();
     let expectedDeploymentString = "\n\nDeployments";
-    const originalTimestamp = +MockFactory.createTestTimestamp();
+    const originalTimestamp = +MockFactory.createTestTimestamp() + 4;
```

Comment:
What's the `+` in front of MockFactory?

Reply:
Parse as a number

Reply:
magic :p

Reply:
woah - really?? Is that a new typescript thing?